x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/90_behavioral_analytics_event_post.yml

```diff
@@ -2,6 +2,7 @@ setup:
   - do:
       search_application.put_behavioral_analytics:
         name: my-test-analytics-collection
+        ignore: 400
```

Comment:
I can't get rid of resource already exists exception when executing multiple tests. I've opted for ignoring problems on creating the collection, as the test will fail in case this is not expected.